core/kernel/impl/class.ApiModelOO.php

```diff
@@ -90,37 +90,24 @@ public function importXmlRdf($targetNameSpace, $fileLocation)
      */
     public function getResourceDescriptionXML($uriResource)
     {
-        $returnValue = (string) '';
-
-
-
-
-
         $dbWrapper = core_kernel_classes_DbWrapper::singleton();
         $subject = $dbWrapper->quote($uriResource);
 
-        $baseNs = [
-                        'xmlns:rdf'     => 'http://www.w3.org/1999/02/22-rdf-syntax-ns#',
-                        'xmlns:rdfs'    => 'http://www.w3.org/2000/01/rdf-schema#'
-                    ];
+        $namespaceCounter = core_kernel_persistence_smoothsql_SmoothModel::DEFAULT_WRITABLE_MODEL;
```

Comment:
Though I finally understood what this variable needed for, could you pls rename it to smth more meaningful.?
Cuz for now it's a bit odd when variable named `...Counter` and it is initialized with `::DEFAULT_WRITABLE_MODEL`.
p.s.
Proposal -> `$baseNamespaceId`. But this will also mean refactoring the part with `$namespaceCounter` usage, like initializing it before `while` loop on line 118, like `$namespaceCounter = $baseNamespaceId` .
OR, as I don't really think, that `DEFAULT_WRITABLE_MODEL` value would be ever changed in CG. So, maybe, for sake of clarity it would be better to just initialize variable with `1`?